### PR TITLE
Make Share URL configurable

### PR DESCRIPTION
**Pull request by anishsheela**
Currently the share URL domain in the profile is not configurable. This creates a new environment variable and remove the hardcoding.

### CODE REVIEW

File: env/env.tsx [hunk 1]
```diff
@@ -5,3 +5,4 @@ export const CHAT_URL = Constants.expoConfig?.extra?.chatUrl ?? 'ws://localhost:
 export const IMAGES_URL = Constants.expoConfig?.extra?.imagesUrl ?? 'http://localhost:9090/s3-mock-bucket';
 export const AUDIO_URL = Constants.expoConfig?.extra?.audioUrl ?? 'http://localhost:9090/s3-mock-audio-bucket';
 export const STATUS_URL = Constants.expoConfig?.extra?.statusUrl ?? 'http://localhost:8080';
+export const WEB_BASE_URL = Constants.expoConfig?.extra?.webUrl ?? 'https://web.duolicious.app';
```
Comment (duogenesis) [inline env/env.tsx:8]:
```suggestion
export const WEB_BASE_URL = Constants.expoConfig?.extra?.webUrl ?? 'http://localhost:8081';
```

Then add `DUO_WEB_BASE_URL: https://web.duolicious.app` here:

https://github.com/duolicious/duolicious-frontend/blob/8de775f6fb75204c17c4cc6b5af4341b781d58b6/.github/workflows/publish.yml#L40-L46

**Review by duogenesis — CHANGES_REQUESTED** [review]
Thanks for improving this. Just one small change then we can merge

File: env/env.tsx (hunk 1 above)
Comment (anishsheela) [reply env/env.tsx]: Resolved

**Comment by anishsheela** [comment]
Pushed with changes. Please review.

**Review by duogenesis — APPROVED** [review]

**Pull request merged**